Description: Fix Sphinx configuration so the documentation can be built
 The Sphinx configuration searches for the pycares version number in
 ../setup.py, which fails as it is no longer here.
 .
 This patch makes it look in the correct place ../pycares/_version.py.
Author: Tanguy Ortolo <tanguy+debian@ortolo.eu>
Bug: https://github.com/saghul/pycares/issues/33
Last-Update: 2016-08-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: pycares/docs/conf.py
===================================================================
--- pycares.orig/docs/conf.py
+++ pycares/docs/conf.py
@@ -14,7 +14,7 @@
 import re
 
 def get_version():
-    return re.search(r"""__version__\s+=\s+(?P<quote>['"])(?P<version>.+?)(?P=quote)""", open('../setup.py').read()).group('version')
+    return re.search(r"""__version__\s+=\s+(?P<quote>['"])(?P<version>.+?)(?P=quote)""", open('../pycares/_version.py').read()).group('version')
 _version = get_version()
 
 # If extensions (or modules to document with autodoc) are in another directory,
